extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -156,6 +156,9 @@ export class AmpSlideScroll extends BaseSlides {
    */
   touchMoveHandler_() {
     this.clearAutoplay();
+    if (!this.hasNativeSnapPoints_) {
+      return;
+    }
```

Comment:
cancel timeout registers a touch move handler - which does some thing - combine it here.